packages/toolpad-core/src/SignInPage/RememberCheckbox.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { FormControlLabel, Checkbox } from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+
+interface RememberCheckboxProps {
```

Comment:
Can't we just allow all `slotProps` from `FormControlLabel` and make sure they're passed to the component?